### PR TITLE
grammar and format fixes

### PR DESCRIPTION
**Pull request by butlerx**

### CODE REVIEW

File: data/schedule.yaml [hunk 1]
```diff
@@ -9,12 +9,13 @@ locations:
...
         description: >
-          Help yourself to a bite to eat and a cup of coffee, courtesy of Bloomberg!
+          Help yourself to a bite to eat and a cup of coffee, courtesy of
+          Bloomberg!
       - name: Introduction
         by: The SISTEM Team
         time: '09:00'
         description: >
-          A welcoming speech to begin the day from some of the organisers
+          A welcoming speech to begin the day with from some of the organisers
```
Comment (CSmartt) [inline data/schedule.yaml:18]: `with from`

**Review by CSmartt — CHANGES_REQUESTED** [review]
One stray word, but all gg besides that. Thanks for taking the time to clean this up :)

File: data/schedule.yaml (hunk 1 above)
Comment (butlerx) [reply data/schedule.yaml:18]: `with from` is correct

Comment (butlerx) [reply data/schedule.yaml:18]: the alternative is `begin the day, from`

Comment (CSmartt) [reply data/schedule.yaml:18]: Brain slow, need coffee. Woops.

**Review by CSmartt — APPROVED** [review]

**Pull request merged**